src/otx/recipe/detection/atss_resnext101_tile.yaml

```diff
@@ -0,0 +1,51 @@
+model:
+  class_path: otx.algo.detection.atss.ResNeXt101ATSS
```

Comment:
this is where the refactoring of develop is not reflected. (all tiling recipes)

Reply:
OK, will check today